ResearchKit/Common/ORKFormItemCell.m

```diff
@@ -493,6 +493,72 @@ - (BOOL)isAccessibilityElement {
 @end
 
 
+#pragma mark - ORKFormItemConfirmTextCell
+
+@implementation ORKFormItemConfirmTextCell
+
+- (void)setSavedAnswers:(NSDictionary *)savedAnswers {
+    [super setSavedAnswers:savedAnswers];
+    
```

Comment:
Decided to throw the exception in the super class, since savedAnswers is available in ORKFormItemCell.